.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,39 @@
+name: Feature request
+description: Create a report to help us improve
+labels: [ "enhancement", "new" ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for submitting a feature request! Would you mind telling us a bit more about what you'd like to see?
+
+  - type: textarea
+    id: description
+    attributes:
+      label: "Describe the feature request"
+      description:  What is the feature you would like to see implemented in Unleash?
+      placeholder:
+    validations:
+      required: true
+
+  - type: textarea
+    id: background
+    attributes:
+      label: Background
+      description: |
+        Why do you want this feature in Unleash?
+        Is it related to a problem of some sort?
+        Help us understand why this feature matters to you.
+        Feel free to include images or any other material that may provide more insight.
+      placeholder:  |
+        A long time ago, in a galaxy far, far away ...
+
+  - type: textarea
+    id: solutions
+    attributes:
+      label: Solution suggestions
+      description: |
+        Have you thought about how we may implement the feature?
+        Have you thought about alternative solutions?
+        Any drawbacks or advantages of the solutions?
+      placeholder: If you twist the knuckleburf and swizzle the huckleshuck while simultaneously ...
```

Tiny bit of extra white space here

```suggestion
      placeholder: If you twist the knuckleburf and swizzle the huckleshuck while simultaneously...
```

---

Ooh, that might be a language thing! I Norwegian you _should_ (I think!) have a space before an ellipsis (which I'm not _technically_ using), but it might be different in English. Investigation time! 🕵🏼

---

Ah, so this is, of course, a style guide thing! As long as we're internally consistent, I'm happy! Imma leave it like this for now and we can fight over it later 🥊